packages/libs/client-utils/src/marmalade/helpers.ts

```diff
@@ -89,3 +95,16 @@ export const formatWebAuthnSigner = (
 
   return formatSingleSigner(signer);
 };
+
+export const isKeysetGuard = (guard: Guard): guard is Keyset => {
```

Comment:
For the sake of it, I'd be more specific with this check, just to avoid any unwanted behaviors.

```
Object.prototype.keys = 'hello'
isKeysetGuard({}) // true
```
So either add `hasOwnProperty` or `Array.isArray(guard.keys)` would prevent this from occuring